packages/2019-transportation/package.json

```diff
@@ -44,6 +44,7 @@
     "d3-array": "^2.3.1",
     "express": "^4.14.1",
     "invariant": "^2.2.2",
+    "moment": "^2.24.0",
```

Comment:
Use [date-fns](https://date-fns.org/) instead. It will be a super easy switch. Make sure to use `yarn remove` to get rid of `moment` so that it cleans up the `yarn.lock` file

Reply:
cool working on this now. date-fns doesn't have support for parsing durations, as in going from a string `mm:ss` to an integer or decimal value of seconds or minutes:

https://github.com/date-fns/date-fns/issues/284

so, now that i think about it, it feels more like something that should be handled on the API side. our views really want to handle numbers more than duration strings.

for now i think i'll write a quick duration parser and then also see what it would take to update the API.